src/detectors/RPOTS/RPOTS.cc

```diff
@@ -8,15 +8,37 @@
 
 #include "extensions/jana/JChainFactoryGeneratorT.h"
```

Comment:
Do we need a separate plugin for rpots and offmtrk?

Reply:
I'm not sure, what would be the arguments for/against combining them?

Reply:
Easier to maintain fewer files and plugins. But we shouldn't put completely unrelated code in one plugin either because then you lose the flexibility of loading one detector chain without needing to load the other. Since the rpots and offmtrk are closely related, having them in one plugin makes sense to me. If there's going to be a next layer that combines results from both, even more so.

Reply:
Don't forget we will be adding a whole new setup for reconstruction, and it will start with the RP first. It makes much more sense while we are actively developing to keep them separate. This algorithm is not a fancy one, and not hard to maintain, anyhow. Information from OMD and RP will not really be used together to reconstruct a particle, except in cases of cross-calibration, but that depends heavily on the vacuum system design.